inc/package.class.php

```diff
@@ -76,8 +76,12 @@ public static function getMenuPicture() {
    public function defineTabs($options = []) {
       $tab = [];
       $this->addDefaultFormTab($tab);
-      $this->addStandardTab('Notepad', $tab, $options);
-      $this->addStandardTab('Log', $tab, $options);
+      $plugin = new Plugin();
+      if ($plugin->isActivated('orion')) {
+         $this->addStandardTab(PluginOrionReport::class, $tab, $options);
+      }
```

Comment:
add a nex tab for package. Maybe it would be a food idea to add the  tab from the register method at the beginning of the PR. I never tried such scenario yet.